opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -311,24 +311,6 @@ enum AggregationTemporality {
   AGGREGATION_TEMPORALITY_CUMULATIVE = 2;
 }
 
-// DataPointFlags is defined as a protobuf 'uint32' type and is to be used as a
-// bit-field representing 32 distinct boolean flags.  Each flag defined in this
-// enum is a bit-mask.  To test the presence of a single flag in the flags of
-// a data point, for example, use an expression like:
-//
-//   (point.flags & FLAG_NO_RECORDED_VALUE) == FLAG_NO_RECORDED_VALUE
-//
-enum DataPointFlags {
-  FLAG_NONE = 0;
-
-  // This DataPoint is valid but has no recorded value.  This value
-  // SHOULD be used to reflect explicitly missing data in a series, as
-  // for an equivalent to the Prometheus "staleness marker".
-  FLAG_NO_RECORDED_VALUE = 1;
```

Comment:
Where did this flag go?

Reply:
I believe the idea is to remove it in favor of comments describing values that can be used in data point flags.  I believe that is a mistake.  Having an enumeration allows for clearly named consistent values and separate documentation of each potential flag.